From 6229aa74fb47af17744d765cc49dbf94d3cefe12 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Wed, 26 Jan 2022 12:18:30 -0800
Subject: x86: Use CHECK_FEATURE_PRESENT on PCONFIG

PCONFIG is a privileged instruction.  Use CHECK_FEATURE_PRESENT, instead
of CHECK_FEATURE_ACTIVE, on PCONFIG in tst-cpu-features-supports.c.

diff --git a/sysdeps/x86/tst-cpu-features-supports.c b/sysdeps/x86/tst-cpu-features-supports.c
index 9f10f02954..11065b642f 100644
--- a/sysdeps/x86/tst-cpu-features-supports.c
+++ b/sysdeps/x86/tst-cpu-features-supports.c
@@ -143,7 +143,7 @@ do_test (int argc, char **argv)
   fails += CHECK_FEATURE_ACTIVE (movdiri, MOVDIRI);
   fails += CHECK_FEATURE_ACTIVE (movdir64b, MOVDIR64B);
   fails += CHECK_FEATURE_ACTIVE (osxsave, OSXSAVE);
-  fails += CHECK_FEATURE_ACTIVE (pconfig, PCONFIG);
+  fails += CHECK_FEATURE_PRESENT (pconfig, PCONFIG);
   fails += CHECK_FEATURE_ACTIVE (pku, PKU);
 #endif
   fails += CHECK_FEATURE_ACTIVE (popcnt, POPCNT);
